### PR TITLE
fix: ansible hostname

### PR DESCRIPTION
**Pull request by pavelpikta**
Fix issue #8

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ base_root: "/opt"
 # service name
 tf_service: "endeca-toolsandframeworks"
 
+# service host
+tf_hostname: "{{ ansible_hostname }}"
```
Comment (tgadiev) [inline defaults/main.yml]:
```suggestion
tf_hostname: "{{ ansible_fqdn }}"
```

File: tasks/update-workbench-password.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
         - name: "Check ToolsAndFrameworks host availability"
           uri:
-            url: "http://{{ ansible_hostname }}:{{ workbench_port }}/ifcr/login"
+            url: "http://{{ tf_hostname }}:{{ workbench_port }}/ifcr/login"
```
Comment (tgadiev) [inline tasks/update-workbench-password.yml:26]: Only http? https not supported?

File: tasks/update-workbench-password.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@
 
         - name: "Change default workbench admin password"
           uri:
-            url: "http://{{ ansible_hostname }}:{{ workbench_port }}\
+            url: "http://{{ tf_hostname }}:{{ workbench_port }}\
```
Comment (tgadiev) [inline tasks/update-workbench-password.yml:36]: Only http? https not supported?

**Review by tgadiev — CHANGES_REQUESTED** [review]

File: tasks/update-workbench-password.yml (hunk 2 above)
Comment (pavelpikta) [reply tasks/update-workbench-password.yml:26]: Need implementation to configure SSL

File: tasks/update-workbench-password.yml (hunk 3 above)
Comment (pavelpikta) [reply tasks/update-workbench-password.yml:36]: Need implementation to configure SSL

**Review by tgadiev — APPROVED** [review]

**Pull request merged**